tools/flang1/utils/prstab/gram.txt

```diff
@@ -931,8 +931,8 @@
 
 <ac item> ::= <expression> |
               <elp> <ac list> , <implied do control> ) |
-	      <expression> : <expression> <opt stride> |
-	      <elp> <ac list> , <expression> )
+              <expression> : <expression> <opt stride> |
```

Comment:
NIT: This is not a change ? or probably unintended ?

Reply:
Thanks for your alertness, @SouraVX , but actually this change was done intentionally. I was sniffing around this piece of code at an early stage of debugging and noticed that the indentation is inconsistent, so I fixed it.